### ISSUE TITLE
Cannot add existing groups

### ISSUE THREAD

**Issue by austinhuang0131** [majn/telegram-purple#546]
```
27366555: RECEIVED OUT-OF-SEQUENCE NOTIFICATION: 0 vs 7387, 513, <private>
```

**Comment by ars3niy** [comment]
> I'm able to send messages but I cannot see them in Adium

Maybe because https://github.com/majn/telegram-purple/blob/323cab7698e3830db797422a2ad2f6bb2678d272/tgp-2prpl.c#L113

**Comment by EionRobb** [comment]
> > I'm able to send messages but I cannot see them in Adium
> 
> Maybe because
> 
> https://github.com/majn/telegram-purple/blob/323cab7698e3830db797422a2ad2f6bb2678d272/tgp-2prpl.c#L113

No, that's for one-to-one IM's and not group chats.

**Comment by ars3niy** [comment]
Right. Might even be libtgl problem then – it wouldn't be unheard of.

**Comment by Noneatme** [comment]
I have setup BitlBee version 3.6-1.1ubuntu1 with telegram-purple 1.4.1, installed from the default ubuntu repos. Adding a telegram account works fine, but I cannot receive any message from anyone, either a user or a group. Sending messages works fine. I am using the Hexchat client.

Can this be part of this problem?

**Comment by foxmajik** [comment]
> NOTE: Outgoing messages will not work in Adium <= 1.6.0, there is no way to print outgoing

This seems like something that could be handled with a functional error message.

I mean, if basic/essential functionality isn't going to work one might want to print an error message to that effect when the client meets the conditions that cause this problem to happen.
